### PR TITLE
Added recipe for traceback2

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

**Comment by pmlandwehr** [comment]
`linecache2` feedstock didn't render for `py34 and osx` for some reason. Re-running CI now; should fix the build.

**Comment by pmlandwehr** [comment]
`linecache2` is now present. Closing and reopening.

**Comment by pmlandwehr** [comment]
Travis CI seems stuck. Closing and reopening.

File: recipes/traceback2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - traceback2
+
+about:
+  home: https://github.com/testing-cabal/traceback2
+  license: PSF 2.0
+  summary: 'Backports of the traceback module'
+
+extra:
+  recipe-maintainers:
+    - pmlandwehr
```
Comment (jakirkham) [inline recipes/traceback2/meta.yaml]: @anguslees, would you be willing to help maintain `traceback2`?

**Comment by jakirkham** [comment]
Toggling to restart build as Travis CI timed out.

File: recipes/traceback2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    - linecache2
+    - six
+
+test:
+  imports:
+    - traceback2
+
+about:
+  home: https://github.com/testing-cabal/traceback2
+  license: PSF 2.0
```
Comment (jakirkham) [inline recipes/traceback2/meta.yaml]: Please add an issue upstream about including a license file in their `sdist`s. Also please xref it here.

Comment (pmlandwehr) [inline recipes/traceback2/meta.yaml]: Done - see [issue 9](https://github.com/testing-cabal/traceback2/issues/9)

File: recipes/traceback2/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/traceback2/meta.yaml]: Please feel free to add @anguslees here, @pmlandwehr, as he proposed such a recipe he was willing to maintain in PR ( https://github.com/conda-forge/staged-recipes/pull/537 ). If I'm wrong about this assumption, please let us know, @anguslees, and we can take you off after.

**Comment by jakirkham** [comment]
Thanks @pmlandwehr. One more thing then am happy to merge.

**Comment by pmlandwehr** [comment]
(ping @jakirkham for merge)

**Comment by jakirkham** [comment]
Thanks @pmlandwehr.

BTW we have been having some issues with Travis CI of late. So if it takes awhile to convert, that might be why.

**Pull request merged**